### ISSUE TITLE
Bad Password? 

### ISSUE THREAD

**Issue by Cris245** [solarkennedy/sensu-training#3]
Hi there! i'm following this course and for some reason when i tried to log into uchiwa i get the error that the password is bad, 

I already check the files and everything seem to be right , any suggestion? 

Thanks!

**Comment by solarkennedy** [comment]
There a number of places where uchiwa is installed in the beginning and intermediate courses. Which one are you on?

My best guess is that the password in the files don't match what uchiwa has in memory. Have you tried restarting uchiwa?

**Comment by Cris245** [comment]
yep, im doing the intermediate, with puppet works fine, but now im trying to do it with Salt, (i used a new vagrant box for this) and just copy-pasty all the sls to ensure i dont have any typo, but nothing
